File: app/src/main/java/com/tahir/go_jek/Database/AppDB.java
package com.tahir.go_jek.Database;



import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.tahir.go_jek.Models.BaseTrending;

@Database(entities = {BaseTrending.class}, version = 1,exportSchema = false)
public abstract class AppDB extends RoomDatabase {


   public abstract ArticlesDao articleDao();
}

File: app/src/main/java/com/tahir/go_jek/Database/AppDB.kt
package com.tahir.go_jek.Database


import androidx.room.Database
import androidx.room.RoomDatabase

import com.tahir.go_jek.Models.BaseTrending

@Database(entities = [BaseTrending::class], version = 1, exportSchema = false)
abstract class AppDB : RoomDatabase() {


    abstract fun articleDao(): ArticlesDao
}
